src/mds/MDCache.cc

```diff
@@ -13256,5 +13302,30 @@ void MDCache::handle_mdsmap(const MDSMap &mdsmap) {
     it = q.erase(it);
     in->maybe_export_pin();
   }
+
+  /* Handle consistent hash ring during cluster resizes */
+  if (mdsmap.get_max_mds() != oldmap.get_max_mds()) {
+        dout(10) << "Max mds is increased. Noted from MDCache side" << dendl;
```

Comment:
this comment needs updated (max_mds may be higher or lower)

Reply:
This and many debug messages need updated. Instead of just noting what happens, say what the code is going to do about it. For example, this would be better:

`dout(10) << "Checking ephemerally pinned directories for re-export due to max_mds change." << dendl;`